koin-projects/examples/multimodule-ktor/module-a/src/main/kotlin/org/koin/samples/multi_module/module_a/moduleA.kt

```diff
@@ -0,0 +1,27 @@
+package org.koin.samples.multi_module.module_a
+
+import io.ktor.application.Application
+import io.ktor.application.call
+import io.ktor.response.respondText
+import io.ktor.routing.get
+import io.ktor.routing.routing
+import org.koin.dsl.bind
+import org.koin.dsl.module
+import org.koin.experimental.builder.single
+import org.koin.ktor.ext.inject
+import org.koin.ktor.ext.koin
+import org.koin.samples.multi_module.common.IService
+
+fun Application.moduleA() {
+    koin {
+        modules(module {
+            single<ModuleAService>() bind IService::class
+        })
+    }
```

Comment:
Finally here, we just need to specify some modules. Do we need to expose the entire Koin DSL here?

Reply:
we could only have `koinModules(...)`

Reply:
Good suggestion with koinModules - I will add this to DSL

Reply:
Added the extension koinModules, but I don’t really like its implementation and therefore marked it as experimental